README.md

```diff
@@ -1,3 +1,29 @@
 # hw01
 
-My name is Ahyeon Cho.
\ No newline at end of file
+# Brief biography 
+## Introduction
+My name is Ahyeon Cho (*Ah* is pronounced like the *o* in the word otter, *yeon* is pronounced y + uh + n). I am from South Korea.
+
+## Interesting things to know about me
+
+1. I am a die-hard **Friends** fan
+2. I do not like *steamed* carrots 
+3. But I do like *fresh* carrots
+4. I memorize almost all **90s disney songs** by heart
+5. The follwing image is what my life has been like since quarantine
+
+![Addiction to Zelda](https://pmcvariety.files.wordpress.com/2018/04/lzbotw_tvc_2.jpg)
+
+## Reasons for taking this course
+I am *very* technologically challenged. I learned much later compared to my peers how to capture an image on my phone. The following are the few things that I am able to do using my computer:
+
+- search on Google
+- read and send e-mail
+- use word and powerpoint 
+- watch Youtube videos
+
```

Comment:
Having a hard time thinking of some critical feedback. Maybe elaborating on which buttons weren't clickable here and saying which earlier task wasn't finished: "it took a long time finding the right buttons, wondering why certain buttons were not clickable and then realizing that I did not finish an earlier task." Maybe also playing around with the size of the different headers, or including an additional link or image (though that wasn't required).

Reply:
I'm curious as well. Was it just the keyboard commands that weren't working for you or trouble with the options on the R interface? I know I had trouble at one point is a menu that listed different "Masters" that I was connected to. I really don't have any other feedback on this project. Great job!